File: Naloga 18/naloga_18.cpp
#include <iostream>

bool deljivost(int x, int y) {
    if ((x % y) == 0) {
        return true;
    }
    return false;
}
void naloga_18() {
    for (int stevilo = 1; stevilo<=300; ++stevilo) {
        if (deljivost(stevilo, 2)) {
            std::cout << stevilo << std::endl;
            continue;
        };
        if (deljivost(stevilo, 7)) {
            std::cout << stevilo << std::endl;
        };
    };
};

int main() {
    naloga_18();
    return 0;
};

File: Naloga 18/naloga_18.py
def naloga_18():
    def deljivost(x, y):
        if x % y == 0:
            return True
    for stevilo in range(1, 300):
        if deljivost(stevilo, 2):
            print(stevilo)
            continue
        if deljivost(stevilo, 7):
            print(stevilo)

if __name__ == '__main__':
    naloga_18()